config/cmake/HDFMacros.cmake

```diff
@@ -369,7 +369,7 @@ macro (HDFTEST_COPY_FILE src dest target)
 endmacro ()
 
 macro (HDF_DIR_PATHS package_prefix)
-  option (HDF5_USE_GNU_DIRS "TRUE to use GNU Coding Standard install directory variables, FALSE to use historical settings" FALSE)
+  option (HDF5_USE_GNU_DIRS "ON to use GNU Coding Standard install directory variables, OFF to use historical settings" OFF)
```

Comment:
There is an equivalent option in ExamplesCache.cmake file.

Reply:
And there is an HDFMacros.cmake file in HDF5Examples/config/cmake folder

Reply:
I have updated these two files.